mcumgr-core/src/main/java/io/runtime/mcumgr/managers/FsManager.java

```diff
@@ -212,13 +212,12 @@ protected UploadResponse write(@NotNull byte[] data, int offset) throws McuMgrEx
      * <p>
      * The upload may be controlled using the {@link TransferController} returned by this method.
      *
-     * @param data The file data to upload.
      * @param callback  Receives callbacks from the upload.
      * @return The object used to control this upload.
      * @see TransferController
      */
     @NotNull
-    public TransferController fileDownload(@NotNull String name, @NotNull byte[] data, @NotNull DownloadCallback callback) {
+    public TransferController fileDownload(@NotNull String name, @NotNull DownloadCallback callback) {
```

Comment:
Deprecate the old method signature and add a new one rather than changing the API to avoid breaking changes.

Reply:
Ok